x/nexus/keeper/migrate.go

```diff
@@ -39,3 +47,23 @@ func deregisterUaxlAsset(ctx sdk.Context, k Keeper) {
 		k.setChainState(ctx, chainState)
 	}
 }
+
+func addModuleParams(ctx sdk.Context, k Keeper) {
+	defaultParams := types.DefaultParams()
+	k.params.Set(ctx, types.KeyChainMaintainerMissingVoteThreshold, defaultParams.ChainMaintainerMissingVoteThreshold)
+	k.params.Set(ctx, types.KeyChainMaintainerIncorrectVoteThreshold, defaultParams.ChainMaintainerIncorrectVoteThreshold)
+	k.params.Set(ctx, types.KeyChainMaintainerCheckWindow, defaultParams.ChainMaintainerCheckWindow)
+}
+
+func migrateChainMaintainers(ctx sdk.Context, k Keeper) error {
```

Comment:
what does this do? does it overwrite the current maintainers or do we also need to clean up the old state?